.gitignore

```diff
@@ -71,3 +71,6 @@ $RECYCLE.BIN
 
 # VIM swap files.
 *.swp
+
+# Rust output.
+/milla/target/*
```

Comment:
Should we make a separate `.gitignore` inside of the `milla` dir to keep things organised?

Reply:
Since it's only one line so far, I don't mind leaving it up here. If it expands later, a separate file would be a good idea.